crates/turbo-tasks-memory/src/viz/graph.rs

```diff
@@ -182,19 +182,46 @@ fn visualize_stats_references_internal<'a>(
 }
 
 fn get_task_label(ty: &TaskType, stats: &TaskStats, max_values: &MaxValues) -> String {
```

Comment:
Can we add some text (to this and the others) that says "run with --full-stats for full stats"?